README.md

```diff
@@ -1 +1,12 @@
-# Hello-world
\ No newline at end of file
+Set up a "Hello Microverse" project
+
+    Description of the project.
+    Understanding the concept of version control and GitHub flow
+
+Built With
+    HTML
+    CSS
+
+    GitHub: @github/Balikuddembe
+    Twitter: @EmmanuelBaliku1
+   
```

Comment:
Kindly use the Readme file from the [Microverse Readme Tempelate](https://github.com/microverseinc/readme-template) as described in the [Project Requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#requirements) and customize it according to your project. 🤗